router/h2/src/main/scala/io/buoyant/router/h2/LocalClassifierStreamStatsFilter.scala

```diff
@@ -0,0 +1,48 @@
+package io.buoyant.router.h2
+
+import com.twitter.finagle.param
+import com.twitter.finagle._
+import com.twitter.finagle.buoyant.h2.{Request, Response}
+import com.twitter.finagle.buoyant.h2.param.H2StreamClassifier
+import io.buoyant.router.{PerDstPathFilter, PerDstPathStatsFilter}
+import io.buoyant.router.context.h2.StreamClassifierCtx
+
+/**
+ * Like [[io.buoyant.router.LocalClassifierStatsFilter]],
+ * but specialized for H2 streams.
+ */
+object LocalClassifierStreamStatsFilter {
+
+  def module: Stackable[ServiceFactory[Request, Response]] =
+    new Stack.Module2[param.Stats, StreamStatsFilter.Param, ServiceFactory[Request, Response]] {
+      val role: Stack.Role = PerDstPathStatsFilter.role
```

Comment:
Should this be `StatsFilter.role`?  Or possibly `LocalClassifierStatsFilter.role`?  (the choice is academic, they have the same value)